### ISSUE TITLE
[BUG] a320 still band left and wont correct itself.  

### ISSUE THREAD

**Issue by Superlister97** [flybywiresim/a32nx#1093]
The aircraft keeps banking left and wont correct itself.... it also wont follow the flight plan I have provided. is there any fix for this issue ? thanks

**Comment by sgtxd** [comment]
I noticed even other aircraft banking left and not always following the flight plan, maybe its a bug at MSFS?

**Comment by Superlister97** [comment]
> I noticed even other aircraft banking left and not always following the flight plan, maybe its a bug at MSFS?

Thats very possible.

**Comment by donstim** [comment]
Increase the deadband for all axes in your controller. It is most likely sending spurious signals that are being interpreted by MSFS as control inputs.

**Comment by Benjozork** [comment]
Please do not remove the issue template.

**Comment by Superlister97** [comment]
> Increase the deadband for all axes in your controller. It is most likely sending spurious signals that are being interpreted by MSFS as control inputs.

its working now ! thank you so much ! :D

**Comment by donstim** [comment]
I have some new information on this topic that may help others with this issue.  I am not sure of the best way to disseminate it, but I thought I would at least add it to this open issue so the mods or whoever reads this issue will be aware of it.

I have 2 controllers, one affected by this issue and one not. I opened the properties for the controllers in the Windows Game Controllers control panel. I noticed that the "cross" showing where the controller is in the x- and y-axes is in the center for the controller with no issue, but off-center for the control that exhibits the issue. I think this is sending an x- and/or y-axis signal that MSFS is accepting as a valid control input.

For my "good" controller, it always returns to a centered output when released. My "bad" controller does not return exactly to center. Until Asobo fixes this controller issue, I suggest those who have this issue calibrate their joysticks each time before firing up the sim and, if their controller does not return exactly to center after release, make sure they have a suitable dead zone set.
